docs/user/library/xml/internal/tutorial.rst

```diff
@@ -46,7 +46,7 @@ First we must configure the pom.xml file of our project to use the `maven-jaxb2-
    
    * http://confluence.highsource.org/display/MJIIP/Sample+projects
    
-   You may wish to download a newer copy of the instructions seem out of date.
+   You may wish to download a newer copy of the instructions if they seem out of date.
```

Comment:
"You may wish to download a newer copy if the instructions seem out of date." would also work, and may have been the original intent.